pkgs/by-name/ov/overskride/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, fetchFromGitHub, stdenv, rustPlatform, cargo, rustc, meson, ninja
+, pkg-config, wrapGAppsHook4, desktop-file-utils, appstream-glib
+, blueprint-compiler, dbus, gtk4, libadwaita, bluez, libpulseaudio }:
+let
+
+  pname = "overskride";
+  version = "0.5.6";
+  owner = "kaii-lb";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-syQzHHT0s15oj8Yl2vhgyXlPI8UxOqIXGDqFeUc/dJQ=";
+  };
+
+in stdenv.mkDerivation {
```

Comment:
```suggestion
in rustPlatform.buildRustPackage {
```

Reply:
Thanks for the tips :pray: 

`rustPlatform.buildRustPackage` fails to build, with a `build.ninja: No such file or directory` error.